ecommerce/models.py

```
@@ -0,0 +1,3 @@
+from django.db import models
```

Ruff (0.8.2)

`django.db.models` imported but unused

Remove unused import: `django.db.models`

(F401)